specification.md

```diff
@@ -127,6 +129,45 @@ Each `identifier` object includes a `system` and a `value`.
 	
 * Additional identifiers: Any number of additional identifiers MAY be included. Each should populate `system` and `value` as appropriate.
 
+#### Each Location object may optionally include the following extension JSON objects in the Location's `extension` array.
+
+* "Location Feature" extension: used to convey one or more features or services for this Location (e.g., for accessibility services)
+
+	| field name | type | description |
+	|---|---|---|
+	|`url`| string | fixed value of `"http://fhir-registry.smarthealthit.org/StructureDefinition/feature"`|
+	|`valueCoding` | JSON object| Coded representation of a location feature or service|
+	| &nbsp;&nbsp;&rarr;&nbsp;`system` | string | See Location Features codings below |
+	| &nbsp;&nbsp;&rarr;&nbsp;`code` | string |  See Location Features codings below |
+	| &nbsp;&nbsp;&rarr;&nbsp;`display` | string | See Location Features codings below |
+	
+  * Location Feature: Wheelchair Access
+  ```
+  {
+    "system": "http://fhir-registry.smarthealthit.org/CodeSystem/feature",
+    "code": "wheelchair-access",
+    "display": "Wheelchair Access"
+  }
+  ```
+  
+  * Location Feature: Drive-through Access
+  ```
+  {
+    "system": "http://fhir-registry.smarthealthit.org/CodeSystem/feature",
+    "code": "drive-through",
+    "display": "Drive-through Access"
+  }
+  ```  
+  
+  * Location Feature: Interpreter Services
+  ```
+  {
+    "system": "http://fhir-registry.smarthealthit.org/CodeSystem/feature",
+    "code": "interpreter",
+    "display": "Interpreter Services"
+  }
+  ```
```

Comment:
Is there a useful way to talk more specifically about which languages here? e.g. Spanish might be supported, but not Tagalog.

Reply:
We could; e.g., we could add another extension for this, with codes from http://hl7.org/fhir/valueset-languages.html. Though given the limited feedback we're getting on this PR, I'm concerned about over-specifying. If you're building an aggregation tool and *have* this kind of information, I'm happy to talk through how to represent it, even if we don't try to spell out all these details in SMART Scheduling Links yet.

Reply:
I’m working with some of the VaccinateCA and similar community groups who are phone-banking to gather detailed info on locations. This is in their list of data to gather, but not sure how frequently or detailed they have it. Will check. I also know Asian/Pacific Islander advocacy groups around San Francisco (where I am) are gathering this info for specific languages.

Maybe useful if this can have specific language values or no set values for both detailed and non-detailed cases.